### PR TITLE
2018-07-02-Lean-Startup

### PR DESCRIPTION
**Pull request by vaalto**
Text

### CODE REVIEW

File: _posts/2018-07-02-blockchain-sharing-economy-autonomes-fahren.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+##Lean Startup Experiment
+
+In der Startup-Welt gehört der Lean Ansatz inzwischen zum Selbstverständnis. In etablierten Unternehmen findet er gerade 
+erst Einzug. Hier bedarf es noch einiger Überzeugungsarbeit, damit Innovationsteams vor der eigentlichen Entwicklungsarbeit 
+mit noch unausgereiften Ideen Experimente am Markt durchführen dürfen. Ein Business Model Canvas im Workshop wird akzeptiert, 
+doch raus aus dem Gebäude? Viele, viele Hürden und Prozessvorschriften sind hier zu überwinden: angefangen von Grundsätzen wie 
+One-Face to the Customer, Corporate Identity, Qualitätssicherung…
+
+Beim ![XCamp](https://xcamp.co/netvis/index) stärken wir allen, die den Lean Weg gehen den Rücken, tauschen uns über Argumente und Praxiserfahrungen aus. Wie
```
Comment (jschirrmacher) [inline _posts/2018-07-02-blockchain-sharing-economy-autonomes-fahren.md]: The exclamation mark means, that this is a picture. We want a normal link here instead.

File: _posts/2018-07-02-blockchain-sharing-economy-autonomes-fahren.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+One-Face to the Customer, Corporate Identity, Qualitätssicherung…
+
+Beim ![XCamp](https://xcamp.co/netvis/index) stärken wir allen, die den Lean Weg gehen den Rücken, tauschen uns über Argumente und Praxiserfahrungen aus. Wie 
+erkläre ich, dass Lean im Kontext von Produktentwicklung etwas anderes heißt als in Produktion und Kundenservice. Nicht 
+Perfektion und Effizienz, sondern Vermeiden von Zeit und Ressourcen in ungeprüfte, abstrakte Entwicklungschancen.
+
+Interessiert an mehr? Hier entlang geht’s zu den Tickets zum ![XCamp18](https://xcamp.co/netvis/tickets?type=corporate)
+
+Eine gute Zusammenfassung gibt der HBR-Artikel von 
+![Steve Blank, 2013: Why the Lean Start-Up Changes Everything](https://hbr.org/2013/05/why-the-lean-start-up-changes-everything)
```
Comment (jschirrmacher) [inline _posts/2018-07-02-blockchain-sharing-economy-autonomes-fahren.md]: Again, here shouldn't be an exclamation mark.

File: _posts/2018-07-02-blockchain-sharing-economy-autonomes-fahren.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+layout:     post
+title:      Lean Startup oder Was haben Blockchain, Sharing Economy und autonomes Fahren gemeinsam?
+authorPage: Christiane Gerigk
+published:  02.07.2018
+image:      Universal_Tile_XCamp18_Thomas-Edison-700x441.png
```
Comment (jschirrmacher) [inline _posts/2018-07-02-blockchain-sharing-economy-autonomes-fahren.md]: The image is missing in the PR. It should be added to the `media` folder, so it should be specified as `media/Universal_Tile_XCamp18_Thomas-Edison-700x441.png` here.
(This is new behavior after my re-structuring)

Comment (jschirrmacher) [reply _posts/2018-07-02-blockchain-sharing-economy-autonomes-fahren.md]: I've merged your other PR into this one, so the image is included now.

File: _posts/2018-07-02-blockchain-sharing-economy-autonomes-fahren.md (hunk 1 above)
Comment (vaalto) [reply _posts/2018-07-02-blockchain-sharing-economy-autonomes-fahren.md]: I again forgot that, sorry!

**Pull request merged**